pathd/path_pcep.h

```diff
@@ -80,10 +81,41 @@
 		}                                                              \
 	} while (0)
 
+struct pcep_config_group_opts {
+	char name[64];
+	char tcp_md5_auth[TCP_MD5SIG_MAXKEYLEN];
+	bool draft07;
+	bool pce_initiated;
+	int keep_alive_seconds;
+	int min_keep_alive_seconds;
+	int max_keep_alive_seconds;
+	int dead_timer_seconds;
+	int min_dead_timer_seconds;
+	int max_dead_timer_seconds;
+	int pcep_request_time_seconds;
+	int state_timeout_inteval_seconds;
+};
```

Comment:
You should add a flag to identify which value was set. Something similar to https://github.com/opensourcerouting/frr/blob/pathd-pcep/pathd/pathd.h#L108

Reply:
Im using the value of 0 to indicate if they have been set or not. A value of 0 is an invalid timer value, so this works well.

Reply:
Maybe for timer values, but not for boolean flags.

Reply:
These boolean flags will always be false unless explicitly configured true by the CLI, in which case I know it has been set.